frontend/packages/dev-console/src/components/add/AddCardSectionSkeleton.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Skeleton } from '@patternfly/react-core';
+import './AddCardSectionSkeleton.scss';
+
+const AddCardSectionSkeleton: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <div className="odc-add-section-skeleton">
+      <Skeleton
+        height="600px"
+        className="odc-add-section-skeleton__background"
+        screenreaderText={t('devconsole~Loading add actions')}
```

Comment:
This is a nice idea, but with a 4 (or n) column layout, this will let the screen reader speak up 4 (or n) times "Loading add actions". This doesn't help here.

Can the MansonryLayout adds a `aria-label` (only) if its loading and shows the LoadingComponent?

Reply:
not sure how we would add the aria label in the masonry component. removed the screenReader instead.